dts/arm/yaml/atmel,sam-uart.yaml

```diff
@@ -0,0 +1,30 @@
+---
+title: SAM Family UART
+id: atmel,sam-uart
+version: 0.1
+
+description: >
+    This binding gives a base representation of the SAM UART
+
+inherits:
+  - !include uart.yaml
+
+properties:
+  - compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "atmel,sam-uart"
+
+  - reg:
```

Comment:
Its for future usage, right now we are just generating defines, we intent to generate code as well.